docs/autopilots/common/dev/changelogs.md

```diff
@@ -1,5 +1,18 @@
 ## Navio Raspbian changelog
 
+*2020-09-22*
+
+- ArduPilot updated:
+    - ArduCopter: 4.0.3
+    - ArduPlane: 4.0.5
+    - ArduSub: 4.0.1
+
+  - Kernel updated to v4.19.27
+  - Preinstalled ROS Noetic (Buster only)
+  - Default board is Navio2
```

Comment:
Maybe we can rewrite this change?

Reply:
@AlexeyBulatov Does it better now? I don't really know how to say it more shortly and clearly